dbt/models/reporting/reporting.vw_pin_school_impact.sql

```diff
@@ -0,0 +1,93 @@
+-- Script that returns the 10 PINs with the highest AV per school district
+-- taxing agency, by year.
+WITH ranking AS (
+    SELECT
+        info.agency_name,
+        info.agency_num,
+        RANK() OVER (
+            PARTITION BY
+                info.agency_name,
+                pin.year
+            ORDER BY pin.av_board DESC
+        ) AS av_board_rank,
+        pin.pin,
+        pin.class,
+        pin.av_mailed,
+        pin.av_certified,
+        pin.av_board,
+        CAST(
+            (
+                pin.av_board - (
+                    pin.exe_homeowner
+                    + pin.exe_senior
+                    + pin.exe_freeze
+                    + pin.exe_longtime_homeowner
+                    + pin.exe_disabled
+                    + pin.exe_vet_returning
+                    + pin.exe_vet_dis_lt50
+                    + pin.exe_vet_dis_50_69
+                    + pin.exe_vet_dis_ge70
+                    + pin.exe_abate
+                )
+            ) * fact.eq_factor_final AS INT
+        ) AS eav,
```

Comment:
Almost! To get taxable EAV, you'll want AV * equalizer, then subtract out exemptions and abatements, because those are in units of EAV.

```
( 
 (pin.av_board * fact.eq_factor_final) 
   - (pin.exe_homeowner
        + pin.exe_senior
        + pin.exe_freeze
        + pin.exe_longtime_homeowner
        + pin.exe_disabled
        + pin.exe_vet_returning
        + pin.exe_vet_dis_lt50
        + pin.exe_vet_dis_50_69
        + pin.exe_vet_dis_ge70
        + pin.exe_abate)
 ) AS eav

```
It might be helpful to rename `eav `to `taxable_eav` to make it clear that this is the FInal EAV, as in the taxable, post-exemption EAV. 

I'd QC it by finding a few PINs with/without exemptions and an abatement, multiplying taxable EAV by the cumulative rate, and seeing if it equals the tax bill. 

I'm sure my parentheses styling is off, but I ain't ashamed.

Reply:
Someday I'll remember this on my own.